README.md

```diff
@@ -112,6 +112,27 @@ install
 
 Please refer to the CMake Manual for more details.
 
+**Windows**:
+Different configurations have different run-time requirements.  The `Debug`
+configuration uses the windows debug C runtime, whereas the `RelWithDebInfo`
+does not.  This may determine which version you can use with your executable.
+
+Some special notes regarding `cmake`:
+- cmake doesn't need the configuration int the `-B` step since it creates
+  all the `*.vcxproj` files at once.
+- for `cmake --build`, **do no** specify `--parallel`, but **do** specify your
+  `--config RelWithDebInfo` or whichever configuration is appropriate.
```

Comment:
Its unfortunate that `--parallel` is just ignored on Windows. I found that you actually need to set a cflag to have parallel compilation like so:

https://github.com/getsentry/sentry-native/blob/06d436096653925e5d7a493e413c9981ce7c5f1c/tests/cmake.py#L138-L141

Though I’m not really sure if there might be a better way here. Documentation is quite bad :-(

Reply:
Right but the point I was making was that if you **do** add --parallel, then the build step will **only** create the `Debug` configuration and ignores `--config`.  Took me half an hour of experimentation last night to figure out what was going on.  If you leave out the `--parallel` flag, then it will build whatever configuration you specify with `--config`

Reply:
ohhh, I think in that case it might be a different manifestation of https://github.com/getsentry/sentry-native/pull/512, which means that cmakes option parsing is broken

Reply:
could be.  Don't know what the best solution is.  Only that I am not an avid cmake user myself and always struggle a bit when using it.  Wanted to help save people from the problems I had :)